code/src/index.css

```diff
@@ -1,13 +1,143 @@
 body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+  "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: rgb(34, 34, 34);
+  color: white;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/* STYLING TEXT */
+
+h1 {
+font-size: 22px;
+padding: 0px 0px 10px 20px;
+}
+
+h2 {
+  font-size: 14px;  
+  font-family: Helvetica;
+  color: #ffffff;
+  margin: 10px 0 5px 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+span {
+  font-size: 14px;  
+  font-family: Helvetica;
+  margin: 0px;
+  color: #a0a0a0;
+}
+
+/* TO INSERT "," OR "&" BETWEEN THE ARTISTS */
+
+.artist > :not(:last-child):not(:first-child):before {
+  content: ", ";
+  white-space: pre;
+  color: #a0a0a0;
+}
+
+.artist > :last-child:before {
+  content: " & ";
+  white-space: pre;
+  color: #a0a0a0;
+}
+
+.artist > :only-child:before {
+  content: "";
+}
+
+
+/* STYLING ALBUM-CARDS */
+
+.albumContainer {
+  display: inline-grid;
+  width: 100vw;
+  gap: 5px;
+  margin: 0px;
+  }
+
+.albumWrapper {
+  width: 300px;
```

Comment:
if width and height is set to percentage instead of px, the album card will scale instead of the gap between the album cards, this might be a possible solution to try instead.